test/labeling/model/test_baseline.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+import numpy as np
+
+from snorkel.labeling.model.baselines import (
+    MajorityClassVoter,
+    MajorityLabelVoter,
+    RandomVoter,
+)
+
+
+class BaselineModelTest(unittest.TestCase):
+    def test_random_vote(self):
+        L = np.array([[1, 2, 1], [0, 4, 3], [3, 0, 0], [1, 2, 2]])
+        rand_voter = RandomVoter()
+        rand_voter.train_model()
```

Comment:
Remove the unnecessary train_model calls? Just for people looking at example usage